libraries/data/doc_generators.py

```diff
@@ -26,6 +27,12 @@ def random_string(length):
     return "".join(random.choice(string.ascii_letters) for _ in xrange(length))
 
 
+def doc_20mb():
+    with open("resources/data/20mb_doc.json") as fh:
```

Comment:
check file exists before open?

Reply:
actually we want tests to fail if doc doesn't exist. So there is no point on doing error checking as we don't want to handle the exception scenario.